scripts/build.js

```diff
@@ -28,10 +28,13 @@ if (glob) {
   if (extraArgs.includes('--lint')) {
     run(`nimbus eslint {packages,plugins}/${glob}/{src,test}`);
   }
-  rimraf.sync(`./{packages,plugins}/${glob}/{lib,esm,tsconfig.tsbuildinfo}`);
-  run(`nimbus babel --clean --workspaces="@superset-ui/${glob}" ${BABEL_CONFIG}`);
-  run(`nimbus babel --clean --workspaces="@superset-ui/${glob}" --esm ${BABEL_CONFIG}`);
-  run(`nimbus typescript --build --workspaces="@superset-ui/${glob}"`);
+  rimraf.sync(
+    `./{packages,plugins}/${glob}/{lib,esm,tsconfig.tsbuildinfo,node_modules/@types/react}`,
+  );
```

Comment:
There's typing conflict when `npm link` installed `@types/react` in sub packages, so we already clean up `@types/react` before build.